components/director/internal/model/webhook.go

```diff
@@ -1,13 +1,20 @@
 package model
 
+import "fmt"
+
 type Webhook struct {
 	ApplicationID string
+	Tenant        string
 	ID            string
 	Type          WebhookType
 	URL           string
 	Auth          *Auth
 }
 
+func (w Webhook) PrettyString() string {
```

Comment:
Good idea, but I don't like this name. We need to highlight that this is used for our internal errors.

I'm also thinking if this is a good time to introduce something like that, because it would be good to discuss, if we need that at all.

Reply:
removed